ops/storage-ddls/cassandra-setup.cql

```diff
@@ -68,6 +70,7 @@ DROP TABLE IF EXISTS sitesettings;
 CREATE TABLE sitesettings(
     sitename text,
     geofence list<double>,
+    places list<place>,
```

Comment:
Thanks for the context. Lets not store the placeids within a bounding box. I dont think its necessary anymore. All we need is the bounding box. Also, the size of the placeids array will likely fail if we're monitoring all of US due to the list constrain in Cassandra.